src/ensembl/src/content/app/browser/browser-track-config/BrowserTrackConfig.tsx

```diff
@@ -154,26 +154,42 @@ export const BrowserTrackConfig = (props: BrowserTrackConfigProps) => {
           </button>
         </dd>
         <dd className={styles.heightSwitcher}>
-          <button className={styles.trackHeightBtn}>
-            <img src={trackHeightBtn} />
-          </button>
+          <div className={styles.trackHeightBtn}>
+            <ImageButton
+              buttonStatus={Status.INACTIVE}
```

Comment:
I suddenly realised that I completely forgot the difference between INACTIVE and DISABLED.

What we want here is a button that is greyed out, does not change cursor on hover, and does not respond to clicks. It's a weird cross between what's currently styled for INACTIVE and for DISABLED.

@imransl do you remember the semantics behind INACTIVE and DISABLED statuses of an image button?

Reply:
IIRC DISABLED is used when we don't want the button to be clickable. Whereas INACTIVE would just grey out the button, but still keep the button in a clickable state.

Reply:
IMO it doesnt make sense to grey out and still being able to click. We should identify possible use cases.

Reply:
I agree. The track height button and the ones onto its right need to be in the `DISABLED` state, since that's what they actually are.